wiki/Tournaments/GBC/GBC_2024_Spring/en.md

```diff
@@ -0,0 +1,221 @@
+   - To be successfully registered, a player must not have violated any [osu! community rules](/wiki/Rules) within the last 12 months.
+   - When registering, the organiser will refer to the player's play counts, heavy derankers and smurfs may be forced to upgrade.
+3. A list of all successfully registered players will be published after the registration phase.
+4. All successfully registered players will be grouped as follows:
+
+| Group | Player's 4K pp |
+| :-: | :-- |
+| EX | 9500 and above |
+| S | 7000 – 8999 |
+| A | 4500 – 6999 |
+| B | 0 – 4499 |
+
+5. If the 4K pp of a player is much higher than the upper limitation of his level, he will be forced to promote into next level. The buffer is *100 pp*.
+6. [Tournament staff members](/wiki/Tournaments/Official_support#staff) must not participate as players in the tournament.
+7. Organisers may force to promote some players that are considered too strong in their levels.
+8. Organisers reserve the right to decline restricted players from participating in the tournament.
+9. It is allowed for players to request to promote their level. They need to send some result screenshotsof next level to the organiser for getting premission.
+
+### Win conditions
+
+1. In the Regular stage, players need to get higher placement in higher level for higher points. The points that can be earned are determined by the placement in every round. The placement according to the average rank for all individual beatmaps in a round.
+2. We have a base points that depends on the placement per round, and a multiplier that depends on the level of players. Here is the calculation method of points.
+
+| Placement | Base points |
+| :-: | --: |
+| 1st place | 20 |
+| 2nd place | 19 |
+| 3rd place | 18 |
+| 4th place | 17 |
+| Top 15% | 16 |
+| Top 22% | 15 |
+| Top 30% | 14 |
+| Top 40% | 13 |
+| Top 50% | 12 |
+| Top 60% | 10 |
+| Top 70% | 8 |
+| Top 80% | 6 |
+| Top 90% | 4 |
+| Top 100% | 2 |
+
+| Level | Multiplier (x is base points) |
+| :-: | --: |
+| Level B | x |
+| Level A | 1.5x + 12 |
+| Level S | 2.3x + 30 |
+| Level EX | 3.4x + 57 |
+
+3. The final qualifier ranking depends on the total amount of points from all 4 rounds. The top 24 players will be able to join the playoff stage.
+4. The top 8 will play in the winners' bracket, while the top 9–24 will play in the losers' bracket.
+5. In case of a tie in the rankings, the player with a lower sum of ranks in every round will take the higher rank.
+6. In the playoff stage, players need to win 6 beatmaps in order to win a match during Quarterfinals and Semifinals, and 7 beatmaps in the Finals and Grand Finals.
+
+### Match procedures
+
+1. A referee will create a multiplayer room 5 minutes before the scheduled match time. Players must join the lobby in that period.
+   - The room settings are `Game mode: osu!mania`, `Team mode: Head-to-Head`, and `Score mode: ScoreV2`. The room name must follow the pattern of `GBC 2024 Spr`.
```

Comment:
`GBC 2024 Spr` looks more like an exact name than a pattern. are all rooms named "GBC 2024 Spr"?

Reply:
it would be like "GBC 2023 Spring: LvEX round1 Sunday10pm"

Reply:
maybe best to have it like this

```suggestion
   - The room settings are `Game mode: osu!mania`, `Team mode: Head-to-Head`, and `Score mode: ScoreV2`. The room name must follow the pattern of the following example: `GBC 2023 Spring: LvEX round1 Sunday10pm`.
```